search.js

```diff
@@ -165,21 +182,30 @@ console.log(table.toString());
 async function run() {
 
   let tryNextPage=true
-  let cursor = null
+  let cursor = CURSOR
   let allResults=[]
 
-  process.stdout.write("\nGathering data ..");
+  process.stdout.write("\nGathering data (timeout=" + TIMEOUT + ") ..");
 
   while(tryNextPage) {
-    process.stdout.write(".");
-    let result = await(doRequest(cursor))
-    if(result.cursor) {
-      cursor = result.cursor
+    let result = await (doRequest(cursor))
+    if (result) {
+      if (result.cursor) {
+        cursor = result.cursor
+      } else {
+        tryNextPage = false
+      }
+      if (LIVERESULTS && result.results.length > 0) {
+        process.stdout.write("\nResults for cursor " + cursor + "\n");
+        drawTable(result.results)
+      } else {
+        process.stdout.write(".");
+      }
+
+      allResults = allResults.concat(result.results)
     } else {
-      tryNextPage = false
+      process.stderr.write("\nSkipping failed request using cursor " + cursor)
     }
-
-    allResults=allResults.concat(result.results)
   }
 
```

Comment:
Another option I'd like to add (but probably won't in this PR) is outputting to a file, especially if there are enough results to fill a screen buffer. It's unpleasant to run this script for a half-hour only to have the results truncated by the terminal buffer. Ask me how I know 😆 😭